src/legacy/core_plugins/embeddable_api/public/np_ready/public/api/utils/bootstrap.ts

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
i find the file/folder organization a bit confusing here ...
this file imo doesn't belong in api folder, its actually using api to register the default actions and triggers.
also, those default actions and trigers live in the lib folder, however they are not libs.
can we create folder actions/triggers/embeddables on the top level and move the bootstrap file there as well ?

Reply:
++ I'm with Peter here, and I think NP team is moving away from generic lib/utils folders and more toward folder by feature/ domain oriented folder structure (discussed in slack channel).

Reply:
Removed `utils` folder and moved `bootstrap()` function to top level.